framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -7945,8 +7946,11 @@ VkDeviceAddress VulkanReplayConsumerBase::OverrideGetBufferDeviceAddress(
     VkDevice                         device       = device_info->handle;
     const VkBufferDeviceAddressInfo* address_info = pInfo->GetPointer();
 
+    // TODO: we get back a noop here due to always encoding KHR flavor
```

Comment:
This sounds like something that should have an issue tracking if you didn't fix it by the end.

Reply:
I confirmed the confusion around vkGetBufferDeviceAddressKHR (w. wrong KHR suffix) to be a bug related to trimming, which was/is present in dev.
created[ this issue](https://github.com/LunarG/gfxreconstruct/issues/1778) to track it and left a TODO with the issue-#

Reply:
Nice.  Thanks!